modules/overview/examples/snip_che-curated-access.adoc

```diff
@@ -1,4 +1,4 @@
 With this setup, you achieve a curated access to {prod-short},
 where cluster administrators control provisioning for each user
 and can explicitly configure various settings including resource limits and quotas.
-Learn more about project provisioning in the link:https://eclipse.dev/che/docs/stable/administration-guide/mounting-a-secret-as-a-file-or-an-environment-variable-into-a-container/#mounting-a-secret-or-a-configmap-as-an-environment-variable-into-a-container[product documentation].
\ No newline at end of file
+Learn more about project provisioning in the xref:administration-guide:provisioning-namespaces-in-advance.adoc[product documentation].
```

Comment:
I noticed that the current link was incorrect. I've updated it to the correct link, and also, I've used `xref:` instead of `link:`

Reply:
```suggestion
Learn more about project provisioning in the xref:provisioning-namespaces-in-advance.adoc[].
```

Reply:
If I apply this change right now, the link does not work, probably because the security best practices doc is not in the administration guide:
![image](https://github.com/user-attachments/assets/f5aa9313-00e9-4cd0-aba8-83869ac0eb42)


Can/should we move the security best practices doc to the administration guide?

Reply:
I see what's happening with the xref (was too focused on the formatting). I don't mind the current set up with the security best practices doc being a standalone but moving it to admin guide would solve the issue I'm currently having with getting it to display properly in downstream docs.

Reply:
@ibuziuk is it ok if I move the security best practices doc to the admin guide?